Gruntfile.js

```diff
@@ -398,57 +416,48 @@ module.exports = function (grunt) {
     'bump-commit'
   ]);
 
-  grunt.registerTask('minify', [
-    'concat',
-    'uglify'
-  ]);
-
   grunt.registerTask('server', [
-    'less:development',
-    'ngconstant',
-    'sails-linker',
+    'precompile',
     'connect:server',
-    'watch:css'
+    'watch'
   ]);
 
-  grunt.registerTask('serverjs', [
-    'less:development',
-    'ngconstant',
-    'sails-linker',
-    'connect:server',
-    'watch:javascript'
+  grunt.registerTask('serverall', [
+    'server'
```

Comment:
Yes they do, I was thinking the same but wasn't sure whether to keep it because we have gotten so used to using `serverall`. But yeah think we should drop `serverall` if your okay with it as well.